assets/keymaps/default-linux.json

```diff
@@ -503,17 +503,17 @@
     }
   },
   {
-    "context": "Editor && inline_completion && !showing_completions",
-    "use_key_equivalents": true,
+    "context": "Editor && inline_completion",
     "bindings": {
-      "tab": "editor::AcceptInlineCompletion"
+      // Changing the modifier currently breaks accepting while you also an LSP completions menu open
+      "alt-enter": "editor::AcceptInlineCompletion"
     }
   },
   {
-    "context": "Editor && inline_completion && showing_completions",
+    "context": "Editor && inline_completion && !showing_completions",
+    "use_key_equivalents": true,
     "bindings": {
-      // Currently, changing this binding breaks the preview behavior
-      "alt-enter": "editor::AcceptInlineCompletion"
+      "tab": "editor::AcceptInlineCompletion"
     }
```

Comment:
The order of the context was intentionally flipped so that we still show `tab` in popovers